tests/accuracy_validation/test_accuracy_validation.py

```diff
@@ -0,0 +1,235 @@
+        process_line(output_line, error_lines)
+
+    err_string = "\n".join(error_lines) if error_lines else None
+    return exit_code, err_string
+
+
+def read_metric(metric_file):
+    metric = 0
+    if os.path.exists(metric_file):
+        input_csv_file = csv.DictReader(open(metric_file))
+        for row in input_csv_file:
+            metric = round(float(row["metric_value"]) * 100, 2)
+    return metric
+
+
+def make_table_row(expected_, key, error_message, metric, diff_target):
+    if metric is not None:
+        row = [str(key), str(expected_), str(metric), str(diff_target), str("-")]
+    else:
+        row = [str(key), str(expected_), str("Not executed"), str("-"), str(error_message)]
+    return row
+
+
+def get_export_test_params(model, model_name, test_id, sample_type, type=None) -> Tuple[str, str, str, str]:
+    workdir = PROJECT_ROOT / model
+    sub_folder = model_name.replace("-", "_")
+    test_folder = str(f"output_export_tests_{sub_folder}.{test_id}")
+    config_dir = workdir / sample_type / model_name / model
+    ir_dir = config_dir / test_folder / 'gpu_export'
+    if type:
+        workdir = PROJECT_ROOT / model.replace('-alt-ssd-export', '')
+        sub_folder = model_name.replace("-", "_")
+        if os.path.isdir(workdir):
+            test_id = str(test_id).replace('_alt_ssd', '')
+        test_folder = str(f"output_export_tests_{sub_folder}.{test_id}")
+        config_dir = workdir / sample_type / model_name / model.replace('-alt-ssd-export', '')
+        ir_dir = config_dir / test_folder / 'gpu_export' / 'alt_ssd_export'
+    return workdir, config_dir, ir_dir, test_id
+
+
+def write_results_table(init_table_string):
+    doc, tag, text = Doc().tagtext()
+    doc.asis('<!DOCTYPE html>')
+    with tag('p'):
+        text('legend: ')
+    with tag('p'):
+        with tag('span', style="Background-color: #{}".format(BG_COLOR_GREEN_HEX)):
+            text('Thresholds Expected are passed')
+    with tag('p'):
+        with tag('span', style="Background-color: #{}".format(BG_COLOR_RED_HEX)):
+            text('Thresholds for Expected are failed, or model was not evaluate')
+    with tag('p'):
+        text('If Reference FP32 value in parentheses, it takes from "target" field of .json file')
+    with tag('table', border="1", cellpadding="5", style="border-collapse: collapse; border: 1px solid;"):
+        with tag('tr'):
+            for i in init_table_string:
+                with tag('td'):
+                    text(i)
+        for key in row_dict:
+            with tag('tr', bgcolor='{}'.format(color_dict[key])):
+                for i in row_dict[key]:
+                    if i is None:
+                        i = '-'
+                    with tag('td'):
+                        text(i)
+    f = open(PROJECT_ROOT / 'results.html', 'w')
+    f.write(doc.getvalue())
+    f.close()
+
+
+eval_config = json.load(open(TEST_ROOT / 'ote_accuracy_validation.json'), object_pairs_hook=OrderedDict)
```

Comment:
Pavel, may be it is better to make a pytest fixture for this block of code?
Probably with `scope="module"` and `autouse=True`

Reply:
P.S. Or without `autouse` may be even better and more clean

Reply:
It's does not works. All fixtures executed after collecting tests, and parameters will not be collected. We can use something about this:
`def get_test_params():
...
return param_list, ids_list


test_param_list, test_ids_list = get_test_params()

@pytest.mark.parametrize("model_, test_id_, sample_type_, model_name_, expected_, alt_export_", param_list,
                         ids=ids_list)
def test_eval(data_dir, model_, test_id_, sample_type_, model_name_, expected_, alt_export_):
...`

Reply:
Done